src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios.md

```diff
@@ -4,14 +4,16 @@ excerpt: ''
 hideFromSidebar: false
 ---
 
-k6 v0.27.0 introduced the concept of scenarios: configurable modes of scheduling VUs
-and iterations, which can be used to model diverse traffic patterns in load tests.
+> ### 🎉 New in v0.27.0
+>
+> This feature is new as of version 0.27.0. Usage of this feature is optional and for the vast majority,
+> existing scripts and configurations will continue to work as before. For a list of breaking changes,
+> see the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).
 
-This is an optional feature and existing scripts and options will continue to work as before
-(with a few minor breaking changes, as mentioned
-in the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).
+## Benefits
 
-There are several benefits of using scenarios in your tests:
+Scenarios allow us to make in-depth configurations to how VUs and iterations are scheduled, allowing
+us to model diverse traffic patterns in our load tests. There are several benefits of using scenarios in your tests:
```

Comment:
- "Scenarios allow us..." and "allowing us" sounds repetitive.
- Nit: the voice of the first sentence is 1st person plural, but the second one switches to 2nd person, which is a bit jarring. I understand we want to make the docs personable, but in many cases keeping an indirect voice is more approachable in technical documentation IMO, and pronouns are messy nowadays 😅 

I would rewrite this to:
```suggestion
k6 scenarios enable configuring how VUs and iterations are scheduled, making it possible to model diverse traffic patterns in load tests. There are several benefits of using scenarios:
```

Reply:
Makes total sense. Thanks!